src/test/java/steps/WorkspaceSteps.java

```diff
@@ -0,0 +1,63 @@
+package steps;
+
+import core.utils.Log;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.apache.log4j.Logger;
+import org.testng.Assert;
+import pivotaltracker.entities.Context;
+import pivotaltracker.entities.Workspace;
+import pivotaltracker.ui.pages.workspace.WorkspacePage;
+import pivotaltracker.ui.pages.workspace.WorkspacePopup;
+import pivotaltracker.ui.pages.workspace.WorkspaceTrackerPage;
+
+/**
+ * WorkspaceSteps class
+ *
+ * @author Limbert Vargas
+ * @version 1.0
+ */
+public class WorkspaceSteps {
+    private Logger logs = Log.getInstance().getLog();
+    private Workspace workspace;
+    private Context context;
+    private WorkspacePage workspacePage;
+    private WorkspacePopup workspacePopup;
+    private WorkspaceTrackerPage workspaceTrackerPage;
+
+    /**
+     * Constructor of workspace steps sending the context.
+     *
+     * @param context init the context.
+     */
+    public WorkspaceSteps(Context context) {
+        this.context = context;
+        this.workspace = context.getWorkspace();
+    }
+
+    /**
+     * Creates a new workspace sending the information.
+     *
+     * @param nameWorkspace contains the workspace values.
+     */
+    @When("I create a new workspace (.*)")
+    public void iCreateANewWorkspace(final String nameWorkspace) {
```

Comment:
you name of method is inconsistent with code conventions of java...

Reply:
I don't understand your observation

Reply:
solve